README.md

```diff
@@ -1,36 +1,31 @@
 ![Build](https://github.com/AdoptOpenJDK/openjdk-api-v3/workflows/Build/badge.svg?branch=master) [![codecov](https://codecov.io/gh/AdoptOpenJDK/openjdk-api-v3/branch/master/graph/badge.svg)](https://codecov.io/gh/AdoptOpenJDK/openjdk-api-v3)
 
-# AdoptOpenJDK API
+# Adoptium API
 
-**NOTICE:** AdoptOpenJDK API v1 has now been removed.
-If you are using v1 please move to the latest version (documented below) as soon as possible.
-
-**NOTICE:** [AdoptOpenJDK API v2](https://github.com/AdoptOpenJDK/openjdk-api/blob/master/README.md) has been deprecated and will be removed.
-If you are using v2 please move to the latest version (documented below) as soon as possible.
+**NOTICE:** [AdoptOpenJDK API v3](https://github.com/AdoptOpenJDK/openjdk-api-v3/blob/master/README.md) has been deprecated as part of the move to Eclipse Foundation and rebrand to Adoptium.
 
 ## Overview
 
-The AdoptOpenJDK API provides a way to consume JSON information about the AdoptOpenJDK releases and nightly builds.  
-Sign up to the [mailing list](https://mail.openjdk.java.net/mailman/listinfo/adoption-discuss) where major API updates will be announced, and visit [adoptopenjdk.net](https://adoptopenjdk.net) to find out more about the community.
+The Adoptium API provides a way to consume JSON information about the Adoptium Temurin releases and nightly builds.  
+Sign up to the [mailing list](https://mail.openjdk.java.net/mailman/listinfo/adoption-discuss) where major API updates will be announced, and visit [adoptium.net](https://adoptium.net) to find out more about the community.
 
 To learn more about how we build & run the API, check out [CONTRIBUTING.md](CONTRIBUTING.md) and the [FAQs](FAQ.md).
 
 ## Usage
 
-The API is documented via swagger.  The swagger documentation can be viewed at: [swagger-ui](https://api.adoptopenjdk.net/swagger-ui). 
-The open api definition for this can be viewed at [openapi](https://api.adoptopenjdk.net/openapi).
+The API is documented using Swagger.  The Swagger documentation can be viewed at: [api.adoptium.net/swagger-ui](https://api.adoptium.net/swagger-ui) 
+The Open API spec for this can be viewed at: [api.adoptium.net/openapi](https://api.adoptium.net/openapi)
 
-For more information, including example queries, please look at [STRUCTURE.md](https://github.com/AdoptOpenJDK/openjdk-api-v3/blob/master/docs/STRUCTURE.md)
+For more information, including example queries, please look at [STRUCTURE.md](docs/STRUCTURE.md)
 
-## Who's using the AdoptOpenJDK API?
+## Who's using the Adoptium API?
 
-The AdoptOpenJDK API has served over 200 million downloads by a wide variety consumers, from individuals to organisations.
+The Adoptium API has served over 200 million downloads by a wide variety consumers, from individuals to organisations.
```

Comment:
may go with something like `Adoptium (and its predecessor at AdoptOpenJDK)` since Adoptium  itself has not done 200m downloads